ecosystem/typescript/sdk/src/generated/core/request.ts

```diff
@@ -102,16 +102,16 @@ const jar = new CookieJar();
 axios.interceptors.response.use((response) => {
   if (Array.isArray(response.headers["set-cookie"])) {
     response.headers["set-cookie"].forEach((c) => {
-      jar.setCookie(new URL(response.config.url), c);
+      jar.setCookie(new URL(response.config.url!), c);
     });
   }
   return response;
 });
 
 axios.interceptors.request.use(function (config) {
-  const cookies = jar.getCookies(new URL(config.url));
+  const cookies = jar.getCookies(new URL(config.url!));
```

Comment:
If you remove the `!` operator, VSCode complains to me, should be pretty easy to repro.

The use of the `!` (bang / non-null assertion operator) at the end of `new URL(config.url!)` causes some issues on typescript compile (just using `tsc`). We should wrap this in an if-statement to make sure it's a string, then set `jar.getCookies` if it is

```ts
../../node_modules/aptos/src/generated/core/request.ts:105:29 - error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type 'string | URL'.

105       jar.setCookie(new URL(response.config.url), c);
```

## Proposed change

```ts
if (config.url) {
  const cookies = jar.getCookies(new URL(config.url));
}
```